spec/dynamoid/persistence_spec.rb

```diff
@@ -339,18 +339,6 @@ def reload_address
     expect(@subscription.send(:dump)[:magazine_ids]).to eq Set[@magazine.hash_key]
   end
 
-  describe 'boolean attributes' do
```

Comment:
Guessing that these tests were also redundant?

Reply:
They are redundant since they implemented the same thing but in a different way and that way is already tested in some other unit tests

Reply:
https://github.com/wishabi/Dynamoid-v2/pull/1/files#diff-686b0e45ce18ac9c53508980fafe31a0R253